db/external_sst_file_ingestion_job.cc

```diff
@@ -469,8 +469,8 @@ Status ExternalSstFileIngestionJob::Run() {
         f.fd.GetNumber(), f.fd.GetPathId(), f.fd.GetFileSize(),
         f.smallest_internal_key, f.largest_internal_key, f.assigned_seqno,
         f.assigned_seqno, false, f.file_temperature, kInvalidBlobFileNumber,
-        oldest_ancester_time, current_time, f.file_checksum,
-        f.file_checksum_func_name, f.unique_id);
+        oldest_ancester_time, current_time, versions_->NewEpochNumber(),
```

Comment:
[Note] I tried out that IngestExternalFile() can ingest multiple files with the same keys without complain. So here I decided to assign different epoch number. Should be fine according the import order as the seqno/file number is assigned by the import order.